FreeRTOS/Demo/CORTEX_MPU_R4F_TI_RM46_HERCULES_GCC/CMakeLists.txt

```diff
@@ -50,19 +50,22 @@ INCLUDE(FetchContent)
 FetchContent_Declare(
     FreeRTOS-Kernel
     GIT_REPOSITORY https://github.com/FreeRTOS/FreeRTOS-Kernel.git
-    GIT_TAG main
+    # Last tested FreeRTOS-Kernel Commit
+    GIT_TAG e8289dfee6e00660b5ad028e9f931ffb76c95840
```

Comment:
Should we not keep it at main to catch regressions?

Reply:
Regressions with mainline would be caught by https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/1018
Idea here is to explicitly state the last commit of the FreeRTOS-Kernel that was tested on-hardware by a team member.